.eslintignore

```diff
@@ -1,4 +1,5 @@
 lib
+examples-lib
```

Comment:
[See comments here](https://github.com/cerner/terra-site/pull/1/files/b76ef6c326444fbaa4b3c22cfc9d4423b4384abf#diff-a374e65eb8ccf18ac88b2919bb4310c0R25) I left here on if we should even generate examples-lib

Reply:
Talked to Emily, I'm good with not having `examples-lib`